### PR TITLE
feat: rename and redirect collections (resolves #1664)

### LINKED ISSUE #1664: Navigation and section titles don't match CMS
## Describe the bug

The site navigation lists "Initiatives, "Events" and "Recount" but the URLs and CMS entries are named "Views", "Initiatives"  and "News" respectively.

## Expected behavior

The CMS interface and URLs should match the sections shown in navigation, e.g. "Initiatives" should be '/initiatives".

## Additional context

Old URLs must be redirected.

### PR DESCRIPTION
**Pull request by greatislander**
* [x] This pull request has been linted by running `npm run lint` without errors
* [x] This pull request has been tested by running `npm run start` and reviewing affected routes
* [x] This pull request has been built by running `npm run build` without errors
* [x] This isn't a duplicate of an existing pull request

## Description

This PR makes the following changes:

- Initiatives are now at `/initiatives` and shown as Initiatives in the CMS
- Events are now at `/events` and shown as Events in the CMS
- Recount is now at `/recount` and shown as Recount in the CMS

All previous URLs are redirected.

## Steps to test

1. Visit deploy preview.
2. Explore Initiatives, Events, and Recount pages.
3. Test filtering and searching.
4. Test the CMS for these content types.
5. Test redirects from known old URLs to new URLs.

**Expected behavior:** Everything works as expected. Old URLs redirect properly.

## Additional information

Not applicable.

## Related issues

Resolves #1664.

### CODE REVIEW

File: src/admin/config.yml [hunk 1]
```diff
@@ -158,16 +134,39 @@ collections:
...
         name: previewImageAltText
         widget: string
         required: false
+  - name: recount
+    label: Recount
+    label_singular: Recount
+    folder: src/collections/recount
+    sortable_fields: [title, date]
+    create: true
+    fields:
```
Comment (chosww) [inline src/admin/config.yml:153]: preview_path is missing for recount, when we had preview_path for news previously.

File: src/utils/get-unique-tags.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ module.exports = collection => {
 	collection.forEach(item => {
 		if (!item.data.tags) {return;}
 		item.data.tags
-			.filter(tag => !["pages", "initiatives", "news", "views", "comments"].includes(tag))
+			.filter(tag => !["pages", "initiatives", "news", "events", "comments"].includes(tag))
```
Comment (chosww) [inline src/utils/get-unique-tags.js:17]: do we need to update news to recount here?

**Review by chosww — COMMENTED** [review]
Thanks @greatislander for the work. I left some comments on parts we may have to fix.

File: src/_includes/layouts/event.njk [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+      <div class="form-field">
+        <span>*</span><h4><label for="name">Name</label></h4><span class="required-name">Required</span>
+        <input id="name" type="text" name="name" placeholder="" aria-label="Enter your name">
+      </div>
+
+      <div class="form-field">
+        <span>*</span><h4><label for="comment">Post a comment or question</label></h4><span class="required-comment">Required</span>
+        <textarea id="comment" name="comment" placeholder="" aria-label="Enter your comment"></textarea>
+        <input type="hidden" name="initiativeId" id="initiativeId" value="{{ id }}">
```
[the remark this answers is not in the document]
Comment (greatislander) [reply src/_includes/layouts/event.njk:32]: @chosww I don't want to do this as those values are used in the Airtable database where comments are written. More broadly I would like to check in with Vera and others about the comments feature. As far as I can tell nobody has ever left a comment on an event page; my preference would be to remove it as that would mean we wouldn't need to maintain the Airtable database connection and related Netlify function.

File: src/admin/config.yml (hunk 1 above)
Comment (greatislander) [reply src/admin/config.yml:153]: News items can't be previewed. They are external links.

File: src/utils/get-unique-tags.js (hunk 2 above)
Comment (greatislander) [reply src/utils/get-unique-tags.js:17]: Maybe, let me check!

File: src/_includes/layouts/event.njk (hunk 3 above)
[the remark this answers is not in the document]
Comment (chosww) [reply src/_includes/layouts/event.njk:32]: Thanks @greatislander, that makes sense.

**Review by chosww — APPROVED** [review]

**Pull request merged**